src/apps/experimental/routes/routes.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { RouteObject, redirect } from 'react-router-dom';
+
+import { REDIRECTS } from 'apps/dashboard/routes/_redirects';
+import { DASHBOARD_APP_PATHS } from 'apps/dashboard/routes/routes';
+import ConnectionRequired from 'components/ConnectionRequired';
+import { toAsyncPageRoute } from 'components/router/AsyncRoute';
+import { toViewManagerPageRoute } from 'components/router/LegacyRoute';
+import { toRedirectRoute } from 'components/router/Redirect';
+import AppLayout from '../AppLayout';
+import { ASYNC_USER_ROUTES } from './asyncRoutes';
+import { LEGACY_PUBLIC_ROUTES, LEGACY_USER_ROUTES } from './legacyRoutes';
+
+export const EXPERIMENTAL_APP_ROUTES: RouteObject[] = [
+    {
+        path: '/',
```

Comment:
as mention on this https://github.com/jellyfin/jellyfin-web/pull/4660#issuecomment-1580741902
```suggestion
        path: '/*',
```

Reply:
Good catch! I didn't notice any warnings about this, but there's quite a bit of logging going on in the console to sift through. I'll update when I resolve conflicts.

Reply:
Updated!